lib/rails-latex/latex_to_pdf.rb

```diff
@@ -81,7 +82,7 @@ def self.generate_pdf(code, config)
     end
 
     # Finish
-    if File.exist?(pdf_file=input.sub(/\.tex$/,'.pdf'))
+    if $?.exitstatus.zero? && File.exist?(pdf_file=input.sub(/\.tex$/,'.pdf'))
```

Comment:
Alternatively, use the "English" library to use the more descriptive `$CHILD_STATUS` variable.

```
$CHILD_STATUS.exitstatus.zero?
```